lib/diagnostic.js

```diff
@@ -38,19 +48,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+      name: 'Skunk',
+      species: 'dog',
+      breed: 'golden retreiver',
+      vocalization: 'woof',
+      vocalize: function(){
+        return 'my name is ' + name + ' and i say ' + vocalization;
+      }
+    };
```

Comment:
Not quite - the `name` and `vocalization` properties aren't visible from inside the `vocalize` function; can you think of a way to reference them?